packages/cli/src/commonlib/cacheAccess.ts

```diff
@@ -144,8 +144,8 @@ export class CryptoCachePlugin {
             }
           }
         }
-      } catch (err) {
-        VsCodeLogInstance.warning(
+      } catch (err: any) {
+        await VsCodeLogInstance.warning(
```

Comment:
Logger APIs are already changed to non-promise in https://github.com/OfficeDev/TeamsFx/pull/9596

Reply:
Thanks. I will merge dev to this branch and try again.

Reply:
Updated. Fix other eslint errors.